wiggum_app/models.py

```diff
@@ -127,6 +137,27 @@ def getDistanceHeatmapDict(df):
 
     return distance_heatmap_dict_list
 
+def replaceTrendDisplayName(cur_result_df):
+    """
+    Add trend display name column to df.
+
+    Parameters
+    -----------
+    cur_result_df : DataFrame
+        The cur_result_df parameter will be passed by filter_df, detect_df and rank_df, not always result_df
+    Returns
+    --------
+    cur_result_df: df appended trend display name
```

Comment:
```suggestion
        A result_df or derivative of it after filtering, detecting or ranking. 
     
    Returns
    --------
    cur_result_df: DataFrame
       input with trend_type moved to the trend_name column and the display name in the trend_type column
```

Reply:
@Shine226 if this text is not the intended behavior change the text *and* the code, because I wrote what the code says